src/index.ts

```diff
@@ -67,8 +67,9 @@ function createGradient(markerCssProperty: string, thickness: number, indentWidt
   return `${gradient} ${startOffset * indentWidth}.5ch/calc(${indentWidth * columns}ch - 1px) no-repeat`
 }
 
-function makeBackgroundCSS(entry: IndentEntry, indentWidth: number, hideFirstIndent: boolean, thickness: number) {
+function makeBackgroundCSS(entry: IndentEntry, indentWidth: number, hideFirstIndent: boolean, thickness: number, activeThickness: number) {
   const { level, active } = entry;
+  activeThickness = activeThickness === -1 ? thickness : activeThickness;
```

Comment:
see above - let's use undefined here and do something like:
```suggestion
  activeThickness = activeThickness ?? thickness;
```

this should handle the case where 0 is passed in as well since that should work as expected

Reply:
Nice. All three done in [d1eb89a](https://github.com/replit/codemirror-indentation-markers/pull/28/commits/d1eb89a0aded2afc6216f37d4207ec19c0f74531).